spyder_kernels/console/kernel.py

```diff
@@ -124,6 +125,18 @@ def set_pdb_ignore_lib(self, state):
         if self._pdb_obj:
             self._pdb_obj.pdb_ignore_lib = state
 
+    def update_syspath(self, path_dict, new_path_dict):
+        """Update the path of the kernel."""
```

Comment:
Please expand this docstring to explain what `path_dict` and `new_path_dict` are.